src/components/ScmRepositoryLink/ScmRepositoryLink.tsx

```diff
@@ -31,6 +34,9 @@ export const ScmRepositoryLink = ({ url, showClipboardCopy = true, showGerritBut
       Gerrit
     </Button>
   );
+  if (repositoryId) {
+    return <Link to={repositoryId} children={parseScmRepositoryTitle({ internalUrl: url })} />;
```

Comment:
why `children` attribute?

Reply:
it was used in [ProjectLink](https://github.com/project-ncl/pnc-web-ui-react/blob/b9ab6a3db06b625f8bcfd2de029848862dfa9793/src/components/ProjectLink/ProjectLink.tsx#L14) component. I just thought this was the preferred way. removed it to the normal way.